content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Migrating Container runtime from dockershim to containerd"
+weight: 10
+content_type: task 
+---
+
```

Comment:
I'd highly recommend following the structure in https://github.com/kubernetes/kubernetes/issues/104878#issuecomment-990909830 as it stays in line with our task template and provides contextual information before diving into the tasks. So a brief overview, prerequisites, etc.

Reply:
Yes the outline will remain the same as you have mentioned and as it is mentioned in the above review here https://github.com/kubernetes/website/pull/30141#discussion_r755274140

Reply:
Please add an introduction to this task.

Also - are there any prerequisites for this task? For example, I assume you need to install containerd. 

If there are please include a "Prerequisites" section before the task steps, i.e. 

```suggestion

This task outlines the steps needed to update your container runtime to containerd from Docker. It is applicable for cluster operators running Kubernetes 1.23 or earlier. 

## Prerequisites 

- Install containerd. For more information see, [containerd's installation documentation](https://containerd.io/docs/getting-started/)
```

Reply:
+1 (and please use `## {{% heading "prerequisites" %}}` for the heading)